src/web/demo/src/main/resources/GeoServerApplication.properties

```diff
@@ -34,6 +34,10 @@ SRSDescriptionPage.name=Name
 SRSDescriptionPage.scope=Scope
 SRSDescriptionPage.remarks=Remarks
 SRSDescriptionPage.areaOfValidity=Area of validity
+SRSDescriptionPage.epsgOrder=EPSG description
+SRSDescriptionPage.epsgOrderDescription=This is the axis order by the official EPSG database, used by the latest OGC specifications. Geographic CRSs along with some projected ones are in North/East order.
```

Comment:
Probably would read better as: "...order from the official EPSG database..." (i.e. "from" instead of "by"